### ISSUE TITLE
Neural net tuning - category 1 classification

### ISSUE THREAD

**Issue by amrdixon** [amrdixon/text_classification#2]
Perform some tuning of the neural net to try to improve performance. Given the drop in performance from the training set (97.4%) to the test set (88.4%), a good start would be to minimize overfitting. One idea to address this would be through regularization (possibly dropout).

**Comment by amrdixon** [comment]
Done! Tuning dropout regularization improved accuracy performance by 1.0%
